tests/ats/test_app.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from typing import List, Tuple
+
+import pytest
+import pykube
+from pytest_helm_charts.clusters import Cluster
+from pytest_helm_charts.k8s.deployment import wait_for_deployments_to_run
+from pytest_helm_charts.k8s.stateful_set import wait_for_stateful_sets_to_run
+
+
+logger = logging.getLogger(__name__)
+
+namespace_name = "loki"
+write_sts_name = "loki-write"
+backend_sts_name = "loki-backend"
+read_deployment_name = "loki-read"
+gateway_deployment_name = "loki-gateway"
+
+timeout: int = 900
+
+@pytest.mark.smoke
+def test_api_working(kube_cluster: Cluster) -> None:
+    """Very minimalistic example of using the [kube_cluster](pytest_helm_charts.fixtures.kube_cluster)
+    fixture to get an instance of [Cluster](pytest_helm_charts.clusters.Cluster) under test
+    and access its [kube_client](pytest_helm_charts.clusters.Cluster.kube_client) property
+    to get access to Kubernetes API of cluster under test.
+    Please refer to [pykube](https://pykube.readthedocs.io/en/latest/api/pykube.html) to get docs
+    for [HTTPClient](https://pykube.readthedocs.io/en/latest/api/pykube.html#pykube.http.HTTPClient).
+    """
+    assert kube_cluster.kube_client is not None
+    assert len(pykube.Node.objects(kube_cluster.kube_client)) >= 1
+
+# scope "module" means this is run only once, for the first test case requesting! It might be tricky
+# if you want to assert this multiple times
+@pytest.fixture(scope="module")
+def ic_components(kube_cluster: Cluster) -> Tuple[List[pykube.Deployment], List[pykube.StatefulSet]]:
```

Comment:
ic is there again 😅

Reply:
I don't know what you're talking about :angel: